cypress/integration/run-processes/run-import-d2cc-process-business-date-view-acceptance-test.js

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright (c) 2022, RTE (http://www.rte-france.com)
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+import * as gc from "../../support/gridcapa-util";
+import * as minio from "../../support/minio";
+import * as ftp from "../../support/ftp-browser.js";
+
+
+const minioUser = Cypress.env('GRIDCAPA_MINIO_USER');
+const minioPassword = Cypress.env('GRIDCAPA_MINIO_PASSWORD')
+const URL = '/cse/import/d2cc'
+const DATE = '2021-09-01'
+const TIME = '22:30'
+const TIMEOUT = 60000;
+Cypress.on('uncaught:exception', (err, runnable) => {
+    // returning false here prevents Cypress from
+    // failing the test
+    return false
+});
+
+
+describe('Check CSE D2CC export corner runs correctly', () => {
+
+    it('Check CSE D2CC export corner runs correctly', () => {
+
+        gc.clearAndVisit(URL)
+        gc.authenticate()
+        gc.getBusinessDateView()
+        gc.setDate(DATE)
+        gc.paginationClickNextButton();
+        //gc.statusForTimestampShouldBe(gc.NOT_CREATED, TIMEOUT, Date.parse(DATE + 'T' + TIME))
```

Comment:
You can keep this check otherwise the previous step is useless.

Reply:
yes, it was commented for debug purpose